9/porting_guide_9.rst

```diff
@@ -84,6 +84,29 @@ Porting custom scripts
 Networking
 ==========
 
+Porting Guide for v9.10.0
+=========================
+
+Breaking Changes
+----------------
+
+community.mysql
+~~~~~~~~~~~~~~~
+
+- collection - support of mysqlclient connector is deprecated - use PyMySQL connector instead! We will stop testing against it in collection version 4.0.0 and remove the related code in 5.0.0 (https://github.com/ansible-collections/community.mysql/issues/654).
+- mysql_info - The ``users_info`` filter returned variable ``plugin_auth_string`` contains the hashed password and it's misleading, it will be removed from community.mysql 4.0.0. Use the `plugin_hash_string` return value instead (https://github.com/ansible-collections/community.mysql/pull/629).
+- mysql_user - the ``user`` alias of the ``name`` argument has been deprecated and will be removed in collection version 5.0.0. Use the ``name`` argument instead.
```

Comment:
These should be in deprecated_features. I opened https://github.com/ansible-collections/community.mysql/pull/679 to fix this, but it's not worth blocking the ansible release over, I don't think.